src/qibolab/instruments/zhinst.py

```diff
@@ -54,10 +54,15 @@ def select_pulse(pulse, pulse_type):
     """Pulse translation"""
 
     if str(pulse.shape) == "Rectangular()":
+        can_compress = True
+        if pulse.type is PulseType.READOUT:
+            can_compress = False
```

Comment:
```suggestion
        can_compress = not pulse.type is PulseType.READOUT
```

Reply:
The name is not my favorite... (sounds more like a sentence than a name)

Remaining close, I would have gone for something like `compressable`. But since this is part of Laboneq, keep it as it is.

Reply:
I agree, but when they already have names for things I just use them,  it saves me the time of thinking them.

Reply:
I perfectly know the feeling, but I also agree on the result for a slightly different reason (keeping the same names creates less variety, and potentially less confusion).